lib/active_force/association/eager_load_projection_builder.rb

```diff
@@ -54,8 +55,8 @@ def apply_association_scope(query)
       # to be pluralized
       def query_with_association_fields
         relationship_name = association.sfdc_association_field
-        query = ActiveQuery.new(association.relation_model, relationship_name)
-        query.fields association.relation_model.fields
```

Comment:
The `.fields` method concats additional fields into the `.select` statement. This is currently duplicating fields on the `.select` statement.

`.select` will mutate the fields being collected, thus removing duplication. This query length reduction will be seen anytime we hit a nested includes on a has_many or has_one object.